src/SlimApplicationFactory.php

```diff
@@ -54,6 +54,12 @@ public function create()
             $this->registerHandlers($app, $configuration['handlers']);
         }
 
+        if (isset($configuration['globalMiddlewares']) && count($configuration['globalMiddlewares']) > 0) {
+            foreach ($configuration['globalMiddlewares'] as $globalMiddleware) {
+                $this->registerGlobalMiddleware($app, $globalMiddleware);
+            }
+        }
```

Comment:
This means that those middlewares are not global in sense of all routes, but all Slim application. Are those called even when the route is not matched?

Could you please document this behaviour in the readme?